@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -1230,6 +1240,129 @@ export class MapMeshStandardMaterial extends THREE.MeshStandardMaterial
     // Mixin declarations end -----------------------------------------------------------
 }
 
+/**
+ * Subclass of THREE.StandardMaterial. Uses the standard material (used in
```

Comment:
Code below sais it's derived from `ShaderMaterial`.

Reply:
good catch